tools/pd-simulator/simulator/cases/import_data.go

```diff
@@ -61,14 +61,14 @@ func newImportData() *Case {
 	simCase.TableNumber = 10
 	// Events description
 	e := &WriteFlowOnSpotDescriptor{}
-	table2 := string(codec.EncodeBytes(codec.GenerateTableKey(2)))
-	table3 := string(codec.EncodeBytes(codec.GenerateTableKey(3)))
+	table12 := string(codec.EncodeBytes(codec.GenerateTableKey(12)))
+	table13 := string(codec.EncodeBytes(codec.GenerateTableKey(13)))
```

Comment:
why change it

Reply:
default the data will generator to table0...9.see the details: https://github.com/tikv/pd/blob/master/tools/pd-simulator/simulator/simutil/key.go#L60-L73(default table count is 10)